### PR TITLE
op-guide: use host volume in Docker command

### PR DESCRIPTION
**Pull request by gyuho**
One should use host volume when running etcd

### CODE REVIEW

File: Documentation/op-guide/container.md [hunk 1]
```diff
@@ -80,13 +80,16 @@ HOST_1=10.20.30.1
 HOST_2=10.20.30.2
 HOST_3=10.20.30.3
 CLUSTER=${NAME_1}=http://${HOST_1}:2380,${NAME_2}=http://${HOST_2}:2380,${NAME_3}=http://${HOST_3}:2380
+DATA_DIR=/tmp/etcd-data
```
Comment (heyitsanthony) [inline Documentation/op-guide/container.md]: `/var/lib/etcd/`?

File: Documentation/op-guide/container.md [hunk 2]
```diff
@@ -70,7 +70,7 @@ In order to expose the etcd API to clients outside of Docker host, use the host
 
 ```
 # For each machine
-ETCD_VERSION=v3.0.0
+ETCD_VERSION=v3.1.0
```
Comment (heyitsanthony) [inline Documentation/op-guide/container.md]: should this use the latest?

**Comment by gyuho** [comment]
@heyitsanthony All addressed. PTAL.

Thanks.

**Comment by heyitsanthony** [comment]
lgtm

**Pull request merged**